.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true,
+    "amd": true
+  },
+  "extends": "airbnb-base/legacy",
+  "globals": {
```

Comment:
Yeah, I was figuring edx-platform would probably have to extend this config with an `.eslintrc` of its own that defines some extra globals. The toolkit and pattern library and new repos we create should be able to just use this as-is though, and I don't think we want to add all of the platform's globals here for that reason.

Jasmine globals are added through the `env: {jasmine: true}` above.